src/main/java/net/neoforged/neoforge/common/extensions/IItemStackExtension.java

```diff
@@ -395,6 +397,13 @@ default <T> T getCapability(ItemCapability<T, @Nullable Void> capability) {
         return capability.getCapability(self(), null);
     }
 
+    /**
+     * @return An immutable {@link ResourceStack} with the current count.
```

Comment:
```suggestion
     * {@return an immutable {@link ResourceStack} with the current count}
```
to also generate a nice javadoc summary for the method.

Reply:
In any case, using exclusively the (usual) `@return` tag should be avoided because it prevents any summary from being rendered. I am fine with just having `Returns whatever.` in the javadoc, i.e. no `@return` and no `{@return}` tag.

Reply:
The use of inline return tags was not banned under Forge, and Neo uses it extensively. A simple search for `{@return` on Neo's GitHub returns over 170 files and many of them have more than one occurrence, so it has hundreds of usages. In any case I maintain my stance from the previous PR thread:
> Regardless, we should not write our documentation based on IDE bugs which ought to be fixed at any time.

Once this bug is fixed by IntelliJ (as far as I'm aware it is not present on Eclipse, it's caused by how IntelliJ parses the file and turns the tokens into the AST structure), do we expect to modify documentation we've introduced to be rewritten to follow Java's guidance on using inline return tags for these cases?

Reply:
Seeing that the returns have been copied as summaries as well I am resolving this.